lib/quicly.c

```diff
@@ -1106,6 +1106,19 @@ static void destroy_all_streams(quicly_conn_t *conn, int err, int including_cryp
     assert(quicly_num_streams(conn) == 0);
 }
 
+int quicly_foreach_stream(quicly_conn_t *conn, void *thunk, int (*cb)(void *thunk, quicly_stream_t *stream))
+{
+    quicly_stream_t *stream;
+    kh_foreach_value(conn->streams, stream, {
```

Comment:
What's `kh_foreach_value`?

Reply:
As the hashtable implementation we use "khash" which is part of [klib](https://github.com/attractivechaos/klib). "kh_" is the prefix that is used for those hash functions, `kh_foreach_value` is the foreach function (or macro, to be precise).